test/parallel/test-zlib-maxOutputLength.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const zlib = require('zlib');
+
+const encoded = Buffer.from('G38A+CXCIrFAIAM=', 'base64');
+
+// Async
+zlib.brotliDecompress(encoded, { maxOutputLength: 64 }, function(err) {
+  assert.ok(err instanceof RangeError);
+});
```

Comment:
```suggestion
zlib.brotliDecompress(encoded, { maxOutputLength: 64 }, common.expectsError({
  code: 'ERR_BUFFER_TOO_LARGE',
  message: 'Cannot create a Buffer larger than ???????? bytes'
}));
```

Reply:
@BridgeAR Suggestion applied in latest commit.